book.toml

```
@@ -1,6 +1,6 @@
 [book]
-title = "Guide to Rustc Development"
-author = "Rustc developers"
+title = "Rust Compiler Development Guide (rustc-dev-guide)"
+author = "The Rust Project Developers"
 description = "A guide to developing rustc"
```

```suggestion
title = "Rust Compiler Development Guide"
author = "The Rust Project Developers"
description = "A guide to developing the Rust compiler (rustc)"
```
I removed `(rustc-dev-guide)` because it felt noisy, for it appears on each rendered page in large font.

---

Makes sense! Applied the suggestion.